main.tf

```diff
@@ -6,15 +6,21 @@ resource "random_uuid" "suffix" {
 
 locals {
   unique_suffix = lower(substr(random_uuid.suffix.id, 0, 5))
-  cluster_name  = "spacelift-cluster-${module.spacelift.unique_suffix}"
+  cluster_name  = coalesce(var.eks_cluster_name, "spacelift-cluster-${module.spacelift.unique_suffix}")
+
+  vpc_id                      = var.create_vpc ? module.spacelift.vpc_id : var.vpc_id
+  private_subnet_ids          = var.create_vpc ? module.spacelift.private_subnet_ids : var.private_subnet_ids
+  public_subnet_ids           = var.create_vpc ? module.spacelift.public_subnet_ids : var.public_subnet_ids
+  server_security_group_id    = var.create_vpc ? module.spacelift.server_security_group_id : var.server_security_group_id
+  drain_security_group_id     = var.create_vpc ? module.spacelift.drain_security_group_id : var.drain_security_group_id
+  scheduler_security_group_id = var.create_vpc ? module.spacelift.scheduler_security_group_id : var.scheduler_security_group_id
 }
 
 module "spacelift" {
   source = "github.com/spacelift-io/terraform-aws-spacelift-selfhosted?ref=adamc%2FCU-8698eh69x-adjustments-for-eks"
 
   unique_suffix = local.unique_suffix
   region        = var.aws_region
-  kms_arn       = var.kms_arn
```

Comment:
why is it removed? 👀

Reply:
Just me overthinking things. I was trying to simplify the module and not expose variables that weren't really needed, but I'll revert it because it doesn't make sense to remove it.